scripts/pre_push_hook.py

```diff
@@ -396,6 +398,16 @@ def main(args=None):
                     python_utils.PRINT(
                         'Push failed, please correct the linting issues above.')
                     sys.exit(1)
+
+            typescript_check_status = 0
+            if does_diff_include_js_or_ts_files(files_to_lint):
```

Comment:
Why we should run a typescript test if js file changes? Any specific reason? (If yes, can you please add a code comment?)

Reply:
Nice Catch! It should be done only for `ts` files. Done!